src/styles/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    margin: 0 auto;
+    margin-top: 33px;
+  }
```

If you specify paddings for "card" instead of "card__content" you can avoid unnecessary margin specification here and the bottom of the content. For example:
```suggestion
  background: #fff;
  
  padding: 33px 16px 16px;

  &__img {
    width: 160px;
  }
```